srcpkgs/julia/template

```diff
@@ -1,65 +1,67 @@
 # Template file for 'julia'
 pkgname=julia
-version=1.4.2
+version=1.5.0
 revision=1
-archs="i686* x86_64*"
+archs="i686* x86_64* armv7l* aarch64*"
 build_style=gnu-makefile
 make_build_args="prefix=/usr sysconfdir=/etc datarootdir=/usr/share
- USE_LLVM_SHLIB=1 USE_BINARYBUILDER=0
- USE_SYSTEM_LIBUV=0 USE_SYSTEM_LIBUNWIND=0 USE_SYSTEM_PATCHELF=1 USE_SYSTEM_LIBM=0
- USE_SYSTEM_DSFMT=0 USE_SYSTEM_LLVM=0 USE_SYSTEM_PCRE=1 USE_SYSTEM_BLAS=0
- USE_SYSTEM_GMP=1 USE_SYSTEM_LIBGIT2=0 USE_SYSTEM_MBEDTLS=1 USE_SYSTEM_LIBSSH2=1
- USE_SYSTEM_CURL=1 USE_SYSTEM_MPFR=1 USE_SYSTEM_SUITESPARSE=0 USE_SYSTEM_UTF8PROC=0
- USE_SYSTEM_ZLIB=1 USE_SYSTEM_P7ZIP=1 USE_SYSTEM_LAPACK=0"
+ USE_BINARYBUILDER=0 USE_SYSTEM_LIBM=0 USE_SYSTEM_DSFMT=0
+ USE_SYSTEM_LIBUV=0 USE_SYSTEM_SUITESPARSE=0
+ USE_SYSTEM_LLVM=1 USE_SYSTEM_LIBUNWIND=1 USE_SYSTEM_PCRE=1
```

Comment:
With using system libunwind, we may be able to get rid of patch. I'll check.

Reply:
I thought about that, but saw at least one line in the patch that reorders linker arguments, so it might still be relevant.